src/main/scripts/org/reaktivity/specification/nukleus/oauth/streams/proxy/reauthorize.inflight.request.same.privileges.update.expiration/accept/server.rpt

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+accept await ROUTED_PROXY
+        "nukleus://streams/oauth#0"
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+accepted
+
+read nukleus:begin.ext ${http:beginEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":scheme", "http")
+                             .header(":method", "GET")
+                             .header(":path", "/path")
+                             .header(":authority", "localhost:8080")
+                             .header("authorization", "Bearer ".concat(oauthTokenOld))
+                             .build()}
+connected
+
+read closed
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":status", "200")
+                              .header("content-length", "11")
+                              .build()}
+write "hello world"
```

Comment:
Do you need the following here, to delay write until after first stream would have expired...
```
write flush
write await BARRIER_NAME
write "hello world"
```
...then trigger the barrier from the test?